README.md

```diff
@@ -1,9 +1,13 @@
 ## Comicbook Web Application
 
 # Installation
-1. First you need node installed on your computer
-2. Then cd into the project directory and run `npm install` to download all the dependencies
-3. Run `gulp` and navigate to `localhost:8080`
+1. Install (HomeBrew)[http://brew.sh/]
+2. Install `nvm` via `brew install nvm` and follow the prompts
+3. Install `node` via `nvm install v6.1.0` -> `nvm use v6.1.0`
+4. Install `mongodb` via `brew install mongodb`
+5. Install `gulp` via `npm install -g gulp`
+6. Install `mocha` via `npm install g mocha`
```

Comment:
If we install `mocha` globally, you can run each individual test file with `mocha filename.js` instead of having to run *all* the tests when you're working on writing a test for individual file.

It's kind of premature to put this here, but I'd rather not have to do this step later when we're trying to debug tests.